salt/cloud/clouds/vmware.py

```diff
@@ -2350,6 +2393,33 @@ def create(vm_):
             )
             clone_spec.customization = custom_spec
 
+        if customization and (devices and 'network' in list(devices.keys())) and 'Windows' in object_ref.config.guestFullName:
+            global_ip = vim.vm.customization.GlobalIPSettings()
+
+            if 'dns_servers' in list(vm_.keys()):
+                global_ip.dnsServerList = vm_['dns_servers']
+
+            identity = vim.vm.customization.Sysprep()
+            identity.guiUnattended = vim.vm.customization.GuiUnattended()
+            identity.guiUnattended.autoLogon = False
+            identity.guiUnattended.password = vim.vm.customization.Password()
+            identity.guiUnattended.password.value = vm_['win_password']
+            identity.guiUnattended.password.plainText = True
+            identity.userData = vim.vm.customization.UserData()
+            hostName = vm_name.split('.')[0]
+            identity.userData.fullName = hostName
+            identity.userData.orgName = "Organization-Name"
+            identity.userData.computerName = vim.vm.customization.FixedName()
+            identity.userData.computerName.name = domain
+            identity.identification = vim.vm.customization.Identification()
+            
```

Comment:
There's trailing whitespace on this line. Lint test will fail if you don't remove the trailing whitespace characters.